sickbeard/name_parser/parser.py

```diff
@@ -121,7 +121,7 @@ def _parse_string(self, name):  # pylint: disable=too-many-locals, too-many-bran
 
         matches = []
         bestResult = None
-        guess = guessit.guessit(name, dict(show_type=self.show_type))
+        guess = guessit.guessit(name, dict(show_type=self.show_type, advanced=True, implicit=True))
         if self.use_guessit:
```

Comment:
Nope. Implicit is already in use and advance will just change what's returned by guessit and it will break the rest of the parser since the other parts are expecting the actual values and not a Match object.